README.md

```diff
@@ -3,6 +3,3 @@ Our main software repository, for both software and firmware.
 
 ### Builds
 `master`: [![Build Status](https://travis-ci.org/UBC-Thunderbots/Software.svg?branch=master)](https://travis-ci.org/UBC-Thunderbots/Software)  
-
-### Coverage
```

Comment:
No more nice icon to show coverage? Can we at least link to the page where results are uploaded?

Reply:
We could, but I don't like the idea. We shouldn't be measuring quality in code coverage. Code coverage can tell you when you _haven't_ tested something (and the CI runs on PR's are great for that) but just because you have coverage doesn't even _remotely_ mean it's tested, and I'm pretty wary about falling into the "I've got 95% coverage so my tests are good trap"

Reply:
Fair enough.